src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2513,14 +2513,10 @@ private SyntaxListBuilder LexConflictMarkerDisabledText(ref SyntaxListBuilder tr
         private void LexConflictMarkerEndOfLine(ref SyntaxListBuilder triviaList)
         {
             this.Start();
-            while (SyntaxFacts.IsNewLine(this.TextWindow.PeekChar()))
```

Comment:
similarly, please move to another PR.  i'm ok merging with the previous commit in that one.

Reply:
Removed.